src/server/common/constants/status-codes.js

```diff
@@ -8,5 +8,6 @@ export const statusCodes = {
   unauthorized: 401,
   forbidden: 403,
   notFound: 404,
-  imATeapot: 418
+  imATeapot: 418,
+  redirect: 302
```

Comment:
I _think_ it's intuitive that this is the temporary redirect (and that we call `301` `permanantRedirect` in the future).  Do you think there's any scope for confusion?

Reply:
my concern over 301 was that it redirects to different places depending on the input (`payload`) and thus I thought it was more a 302 than a 301

Also looking at [MDN 302](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/302) = Found, not temp (which was news to me, but why i chose it over 300)

Reply:
@DrogoNevets  - yup, 100% agreed that 302 is the right choice here.  It's just about the wording of the status code in the map.

Reply:
oh i see, can change it to `Found` as per MDN?